source/includes/_authenticated_api_partnerships.md

```diff
@@ -84,5 +84,85 @@ Find information about a partnership by URL slug.
 
 `GET /api/v1/partnerships/fight-fascism`
 
+### List Petitions
+
+> GET response Body
+
+```json
+{
+  "partnership": {
+    "title": "Fight Fascism",
+    "slug": "fight-fascism"
+  },
+  "petitions": [
+    {
+      "id": 123,
+      "slug": "no-taxes-on-tea",
+      "title": "No Taxes on Tea",
+      ...
+    },
+    {
+      "id": 124,
+      "slug": "stop-burning-coal-1",
+      "title": "Stop Burning Coal",
+      ...
+    },
+    ...
+  ],
+
+  "meta": {
+    "current_page": 1,
+    "total_pages": 12,
+    "previous_page": null,
+    "next_page": 2
+  }
+}
+```
+
+Retrieves a paginated list of a partnership's petitions.
+
+`GET /api/v1/partnerships/fight-fascism/petitions`
+
+
+
+### List Events
+
+> GET response Body
+
+```json
+{
+  "partnership": {
+    "title": "Fight Fascism",
+    "slug": "fight-fascism"
+  },
+  "events": [
+    {
+      "id": 123,
+      "slug": "no-taxes-on-tea",
```

Comment:
To reduce confusion, should we use different example event names from our example petition names? Something that sounds more "event-y"?